examples/Hermitian_Phasing.py

```diff
@@ -0,0 +1,76 @@
+from pyspecdata import *
+from pyspecProcScripts import *
+from pylab import *
+import sympy as s
+from collections import OrderedDict
+from numpy.random import normal, seed
+
+seed(2021)
+rcParams["image.aspect"] = "auto"  # needed for sphinx gallery
+
+t2, td, vd, power, ph1, ph2 = s.symbols("t2 td vd power ph1 ph2")
+echo_time = 10e-3
+f_range = (-400, 400)
+
+with figlist_var() as fl:
+    for expression, orderedDict, signal_pathway, indirect, label in [
+        (
+            (
+                23
+                * (1 - 2 * s.exp(-vd / 0.2))
+                * s.exp(+1j * 2 * s.pi * 100 * (t2) - abs(t2) * 50 * s.pi)
+            ),
+            [
+                ("vd", nddata(r_[0:1:40j], "vd")),
+                ("ph1", nddata(r_[0:4] / 4.0, "ph1")),
+                ("ph2", nddata(r_[0, 2] / 4.0, "ph2")),
+                ("t2", nddata(r_[0:0.2:256j] - echo_time, "t2")),
+            ],
+            {"ph1": 0, "ph2": 1},
+            "vd",
+            "IR",
+        ),
+        (
+            (
+                23
+                * (1 - (32 * power / (0.25 + power)) * 150e-6 * 659.33)
+                * s.exp(+1j * 2 * s.pi * 100 * (t2) - abs(t2) * 50 * s.pi)
+            ),
+            [
+                ("power", nddata(r_[0:4:25j], "power")),
+                ("ph1", nddata(r_[0:4] / 4.0, "ph1")),
+                ("t2", nddata(r_[0:0.2:256j] - echo_time, "t2")),
+            ],
+            {"ph1": 1},
+            "power",
+            "enhancement",
+        ),
+    ]:
+        fl.basename = label
+        data = fake_data(expression, OrderedDict(orderedDict), signal_pathway)
```

Comment:
again, you were not on the most recent version of master -- this seems to have been forked off of da6b0dcc4b0b4727761fb48063ba19cda2a1b9d7, which is way back from 6/11 --> maybe you hadn't pulled master before making this branch?  Anyways, I merged in master to get it to work

Reply:
thank you I really don't know what's going on with my master because as shown on slack I am doing every step you say (pull from master make new branch off master yadda yadda)

Reply:
so, I think we fixed the issue with master now?

Reply:
yes all good!